ocular/src/chain/client/tx.rs

```diff
@@ -18,70 +20,60 @@ const TX_LOGGING_DIR: &str = "/.ocular/logs/txs";
 /// Unix permissions for dir
 const TX_LOGGING_DIR_PERMISSIONS: u32 = 0o700;
 
-/// Metadata wrapper for transactions
-#[derive(Clone, Debug)]
-pub struct TxMetadata {
-    pub chain_id: Id,
-    pub account_number: u64,
-    pub sequence_number: u64,
-    pub gas_fee: Coin,
-    pub gas_limit: u64,
-    pub timeout_height: u32,
-    pub memo: String,
-}
-
-///  Type to hold all information around an account.
-pub struct Account {
-    pub id: AccountId,
-    pub public_key: PublicKey,
-    pub private_key: SigningKey,
-}
-
 impl ChainClient {
+    pub async fn get_basic_tx_metadata(&self) -> Result<TxMetadata, ChainClientError> {
+        let current_height = self.query_latest_height().await?;
+        let timeout_height: u32 = (current_height + 30) as u32;
+        let fee = self.config.default_fee.clone();
+
+        Ok(TxMetadata {
+            fee,
+            gas_limit: 200000,
+            timeout_height,
+            memo: String::default(),
```

Comment:
I would say that there really isn't an appropriate default value other than empty. This method is just to make it less annoying to do a quick tx operation. Even in the event that they need a memo (pretty rare, only for CEXs generally) it seems safe to assume that the user is a power user if they are using a library instead of a wallet product to me? This is a library that will be in lots of different apps, not necessarily CLIs, so expecting user confirmation won't work.

Reply:
I guess the concern would be this scenario: We do our airdrop, and someone wants their tokens sent to a CEX, so they provide some address (not entirely sure how CEX transfers look but I imagine they still have typical looking addresses), and we sent them their tokens, which are then dropped. Even if the user is a power user, if the receiver is less knowledgable of the space, this could prove problematic. Im not entirely sure of the best solution to this kind of situation. What do you think?